dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo

```diff
@@ -44,13 +44,11 @@ model PIAntiWindUpTable "Proportional Integrator with anti-windup and table-base
   parameter Real Y0 "Start value of output";
 
 equation
-  when time >= 0 and u.value <> pre(u.value) then
+  if time >= 0 then
```

Comment:
Have you checked that y.value is actualized each time combiTable1D.y[1] changes ?